src/Form.js

```diff
@@ -117,7 +117,7 @@ class Form {
             if (e.button !== 0) return
 
             if (this.shouldOpenTooltip(e, e.target)) {
-                if (isEventWithinDax(e, e.target)) {
+                if (isEventWithinDax(e, e.target) || (isDDGApp && !isApp)) {
```

Comment:
Mobile apps have a different way of handling tooltip invocation, so we're introducing this exception to solve https://app.asana.com/0/1198567644089701/1200405239350391/f.